clients/src/main/java/org/apache/kafka/clients/producer/KafkaProducer.java

```diff
@@ -1383,9 +1386,13 @@ private ClusterResourceListeners configureClusterResourceListeners(Serializer<K>
      * can be used (the partition is then calculated by built-in
      * partitioning logic).
      */
-    private int partition(ProducerRecord<K, V> record, byte[] serializedKey, byte[] serializedValue, Cluster cluster) {
```

Comment:
Lest my eyes deceive me, this is just formatting, correct?

Reply:
Yes, I have already rolled back this change.